fastlane/lib/fastlane.rb

```diff
@@ -19,6 +20,7 @@
 require 'fastlane/documentation/docs_generator'
 require 'fastlane/other_action'
 require 'fastlane/plugins/plugins'
+require 'fastlane/setup/crashlytics_beta'
```

Comment:
I'd move that to https://github.com/fastlane/fastlane/pull/5201/files#diff-87e6d7e2cbe2a0420c4c4efb74cd3cbcR62, as long as it's a feature flag at least 👍